packages/wallet/src/redux/__tests__/helpers.ts

```diff
@@ -95,3 +95,24 @@ export const itIncreasesTurnNumBy = (
     }
   });
 };
+
+export const itDispatchesThisAction = (action, state: { outboxState?: OutboxState }) => {
+  it(`dispatches ${action.type || 'this action'}`, () => {
+    // The actionOutbox should only dispatch internal actions
+    if (action.type) {
+      // We were passed the whole action
+      expect(action.type).toMatch('WALLET.INTERNAL');
+      expect(state.outboxState!.actionOutbox).toMatchObject(action);
+    } else {
+      // We were just passed the type
+      expect(action).toMatch('WALLET.INTERNAL');
+      expect(state.outboxState!.actionOutbox!.type).toEqual(action);
+    }
+  });
+};
+
+export const itDispatchesNoAction = (state: { outboxState?: OutboxState }) => {
+  it(`dispatches no action`, () => {
+    expect(state.outboxState!.actionOutbox).toBeUndefined();
```

Comment:
Maybe we should check if `outboxState` is defined before asserting that `actionOutbox` property is undefined?